packages/pluggableWidgets/video-player-web/package.json

```diff
@@ -13,6 +13,10 @@
     "mendixHost": "http://localhost:8080/",
     "developmentPort": "3000"
   },
+  "marketplace": {
+    "studioProMinimumVersion": "8.0.0",
```

Comment:
This property might be relevant for more scripts in the future, like automated e2e tests. Therefore, I think we should make this a property on root level and rename it to `minimumMXVersion` or something like that.

Reply:
Fixed

Reply:
I see you renamed the key. Nice! I was also thinking of removing the marketplace property and just having the two child properties as a direct child of the JSON object. So on the same level as `license` and `script`. I think this is better, because this property might be relevant for more scripts in the future, like automated e2e tests.

Reply:
I would keep it completely separated, as people will start to copy and we dont want to ship it with the generator.